{{ cookiecutter.repo_name }}/pyproject.toml

```diff
@@ -1,6 +1,24 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = {{ cookiecutter.module_name|tojson }}
+version = "0.0.1"
+description = {{ cookiecutter.description|tojson }}
+authors = [
+  { name = {{ cookiecutter.author_name|tojson }} },
+]
+{% if cookiecutter.open_source_license != 'No license file' %}license = { file = "LICENSE" }{% endif %}
+readme = "README.md"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    {% if cookiecutter.open_source_license == 'MIT' %}"License :: OSI Approved :: MIT License"{% elif cookiecutter.open_source_license == 'BSD-3-Clause' %}"License :: OSI Approved :: BSD License"{% endif %}
+]
+requires-python = ">={{ cookiecutter.python_version_number }}"
```

Comment:
Since the template usually starts as projects/applications (rather than library code), I think we want to have the Python version as a strict pin for reproducibility.

Reply:
Agreed, that makes sense.